lib/rubocop/cop/lint/ambiguous_block_association.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for ambiguous block association with method
+      # when param passed without parentheses.
+      #
+      # @example
+      #
+      #   # bad
+      #
+      #   # It's ambiguous because there is no parentheses around `a` param
```

Comment:
@bbatsov many cops in `lint` folder have this empty line between `#good` or `#bad` and explanation. I thought it's sort of standard. Ok, will remove description for bad and empty lines between. Thanks.